WooCommerce/Classes/ViewRelated/Products/Variations/ProductVariationsViewController.swift

```diff
@@ -305,7 +305,8 @@ extension ProductVariationsViewController: UITableViewDelegate {
                                                            productImageActionHandler: productImageActionHandler,
                                                            currency: currency,
                                                            presentationStyle: .navigationStack,
-                                                           isEditProductsRelease3Enabled: isEditProductsRelease3Enabled)
+                                                           isEditProductsRelease3Enabled: isEditProductsRelease3Enabled,
+                                                           isEditProductsRelease5Enabled: false)
```

Comment:
Why is `false` here? 🤔  The edit product release 5 will not include only downloadable files.

Reply:
Product variation doesn't have any downloadable file that's why I thought it would be false. But yes, you are right.